components/calculator.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+
+export default () => {
+  const [firstOperand, setFirstOperand] = useState('')
+  const [secondOperand, setSecondOperand] = useState('')
+  const [answer, setAnswer] = useState('')
+  const [operation, setOperation] = useState('')
+  const [equalClicked, setEqualClicked] = useState(false)
+
+  const updateFirstOperand = (event) => {
+    setFirstOperand(event.target.value)
+  }
+
+  const updateSecondOperand = (event) => {
+    setSecondOperand(event.target.value)
+  }
+
+  const updateOperation = (event) => {
+    setOperation(event.target.value)
+  }
+
+  const calculateAnswer = () => {
+    setEqualClicked(true)
+    const firstNumber = parseInt(firstOperand, 10)
+    const secondNumber = parseInt(secondOperand, 10)
+
+    switch (operation) {
+      case 'addition':
+        setAnswer(firstNumber + secondNumber)
+        break
+
+      case 'subtraction':
+        setAnswer(firstNumber - secondNumber)
+        break
+
+      case 'multiplication':
+        setAnswer(firstNumber * secondNumber)
+        break
+
+      case 'division':
+        setAnswer(firstNumber / secondNumber)
+        break
+
+      default:
+        setAnswer('')
+    }
+  }
+  const isNumber = (a) => {
+    if (isNaN(a)) { // eslint-disable-line no-restricted-globals
+      return false
+    }
+    return true
```

Comment:
Don't use an `if` when you are returning true/false like this, just return the result.